### ISSUE TITLE
Botpress stage 1: converse API

### ISSUE THREAD

**Issue by smashah** [open-wa/wa-automate-nodejs#1932]

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i -D`);
exec(`npm run release-ci minor`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [4.12.0](https://github.com/open-wa/wa-automate-nodejs/compare/4.11.1...4.12.0) (2021-07-23)

- build(deps): bump pino from 6.12.0 to 6.13.0 [`#1928`](https://github.com/open-wa/wa-automate-nodejs/pull/1928)
- build(deps-dev): bump @types/node from 16.3.3 to 16.4.1 [`#1930`](https://github.com/open-wa/wa-automate-nodejs/pull/1930)
- build(deps): bump ts-json-schema-generator from 0.94.1 to 0.95.0 [`#1931`](https://github.com/open-wa/wa-automate-nodejs/pull/1931)
- build(deps-dev): bump @typescript-eslint/eslint-plugin from 4.28.3 to 4.28.4 [`#1920`](https://github.com/open-wa/wa-automate-nodejs/pull/1920)
- add BUTTONS_RESPONSE to MessageTypes [`#1914`](https://github.com/open-wa/wa-automate-nodejs/pull/1914)
- build(deps-dev): bump @types/node from 16.3.2 to 16.3.3 [`#1917`](https://github.com/open-wa/wa-automate-nodejs/pull/1917)
- build(deps-dev): bump eslint from 7.30.0 to 7.31.0 [`#1918`](https://github.com/open-wa/wa-automate-nodejs/pull/1918)
- build(deps-dev): bump @typescript-eslint/parser from 4.28.3 to 4.28.4 [`#1921`](https://github.com/open-wa/wa-automate-nodejs/pull/1921)
- Release 4.12.0 [`c9b7f66`](https://github.com/open-wa/wa-automate-nodejs/commit/c9b7f66399e211f86684b879bc0bef7fbbc6fecb)
- 📝  fix docs links #1929 [`b6b1cdb`](https://github.com/open-wa/wa-automate-nodejs/commit/b6b1cdb84b2ee7671ae0b750963f0670ace39ced)
- ✨ CLI: `--bot-press-url/-b` connect your botpress to open-wa via CONVERSE API #1932 [`d790d22`](https://github.com/open-wa/wa-automate-nodejs/commit/d790d2239f80df6f61c3d5c3a6ba24115870f15a)
- 🚧  WIP/ALPHA: prelim multi-device support. Sessions will NOT restart! #1923 [`c7eb7a2`](https://github.com/open-wa/wa-automate-nodejs/commit/c7eb7a2fe04275998bea06d7f5f41e2409339635)
- ✨ feat [INSIDERS]: `client.B` Send a custom Baileys payload #1927 [`3ef8714`](https://github.com/open-wa/wa-automate-nodejs/commit/3ef8714db60c366d94c38acbb28353434d4ff316)
- 🏷️  fix `ghostforward` return type #1924 [`12edda5`](https://github.com/open-wa/wa-automate-nodejs/commit/12edda53b15f2a947842d864c6ad3a1044b8ba44)
- 🏷️  button typing [`a7ba60e`](https://github.com/open-wa/wa-automate-nodejs/commit/a7ba60e9dd0e2d85480c6e3f9653b275a0aa9ec6)
- 🏷️  fix types [`5b7f240`](https://github.com/open-wa/wa-automate-nodejs/commit/5b7f2406f7d52a538ffd85c3513a24c666103d45)
- 🐛 `title` in `sendButtons` is not optional, default to empty string [`f396cba`](https://github.com/open-wa/wa-automate-nodejs/commit/f396cba9ed9abbd2b3dc35062c9dd18d9a430108)
- 💫 GA: `onMessageDeleted` [`dec835f`](https://github.com/open-wa/wa-automate-nodejs/commit/dec835fde576a6f8446f1c9977fc06038c39cde0)

**Comment by smashah** [comment]
```bash
> npx @open-wa/wa-automate@latest -b http://localhost:3000/api/v1/bots/test/converse    
```
